http/logical.go

```diff
@@ -58,10 +99,14 @@ func buildLogicalRequestNoAuth(perfStandby bool, w http.ResponseWriter, r *http.
 		op = logical.DeleteOperation
 		data = parseQuery(r.URL.Query())
 	case "GET":
+		var (
```

Comment:
Nit: Within functions, you usually want var declarations as close to the usage as possible to keep its context close so might be good to move it back down.